docs/modules/publisher.md

```diff
@@ -121,24 +126,205 @@ A basic configuration file looks like this:
+| `EndpointUrl`                    | yes       | String  | N/A                       | The OPC UA Server’s endpoint URL                                                             |
+| `UseSecurity`                    | no        | Boolean | false                     | Desired opc session security mode                                                            |
+| `OpcAuthenticationMode`          | no        | Enum    | Anonymous                 | Enum to specify the session authentication.<br>Options: Anonymous, UserName                  |
+| `UserName`                       | no        | String  | null                      | The username for the session authentication<br>Mandatory if OpcAuthentication mode is UserName|
+| `Password`                       | no        | String  | null                      | The password for the session authentication<br>Mandatory if OpcAuthentication mode is UserName|
+| `DataSetWriterGroup`             | no        | String  | EndpointUrl_#             | The writer group collecting datasets defined for a certain <br>endpoint uniquely identified by the above attributes. <br>This shall be used to identify the session opened into the <br>server. The default value consists of the EndpointUrl string, <br>followed by a deterministic hash composed of the <br>EndpointUrl, security and authentication attributes.|
+| `DataSetWriterId`                | no        | String  | DataSetPublishingInterval | The unique identifier for a data set writer used to collect <br>opc nodes to be semantically grouped and published with <br>a same publishing interval. <br>When not specified a string representing the common <br>publishing interval of the nodes in the data set collection. <br>This the DataSetWriterId shall uniquely identify a data set <br>within a DataSetGroup. The unicity shall be determined <br>using the provided DataSetWriterId and the publishing <br>interval of the grouped OpcNodes.  An individual <br>subscription shall be created for each DataSetWriterId|
+| `DataSetPublishingInterval`      | no        | Integer | false                     | The publishing interval used for a grouped set of nodes <br>under a certain DataSetWriter. When defined it shall <br>override the OpcPublishingInterval value in the OpcNodes <br>if grouped underneath a DataSetWriter. |
+| `Tag`                            | no        | String  | empty                     | TODO                                                                                         |
+| `OpcNodes`                       | yes*      | OpcNode | empty                     | The DataSet collection grouping the nodes to be published for <br>the specific DataSetWriter defined above. |
+
+*Note*: *OpcNodes field is mandatory for PublishNodes_V1, UnpublishNodes_V1 and AddOrUpdateNodes_V1. It shall not be specified for the rest of the methods.
```

Comment:
Please rename `AddOrUpdateNodes_v1` to `AddOrUpdateEndpoints_V1`.

Reply:
Also, what is `*` in front of `OpcNodes` intended for ? Seems like it is missing a closing `*` if you had italic in mind, or double `*`s if you wanted it to be bold.

Reply:
Ah, ok, got it, `*` is an explanation for `yes*` above. Please ignore the comment about `*`

Reply:
right